.github/workflows/ci.yml

```
@@ -1399,13 +1399,17 @@ jobs:
         needs: [ changes ]
-        if: |
-            (needs.changes.outputs.code == 'true' ||
-             needs.changes.outputs.ndepend == 'true' ||
-             github.event_name == 'schedule' ||
-             github.event_name == 'workflow_dispatch') &&
-            (github.event_name != 'pull_request' ||
-             github.event.pull_request.head.repo.full_name == github.repository)
+        # TODO(ndepend-outage): vendor download https://www.codergears.com/protected/GitHubActionAnalyzer.zip
+        # 301-redirects to the homepage, so ndepend-action cannot fetch its engine (fails before any analysis).
+        # Re-enable the condition below once the vendor fixes the URL.
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)